python/src/v1.9/containers/Axes.python.cpp

```diff
@@ -36,12 +40,10 @@ void wrapAxes(python::module &module)
       .def(
          python::init<
             const std::optional<UTF8Text> &,
-            const std::optional<std::vector<containers::Axis>> &,
-            const std::optional<std::vector<containers::Grid>> &
+            const std::vector<VARIANT> &
          >(),
          python::arg("href") = std::nullopt,
-         python::arg("axis") = std::nullopt,
-         python::arg("grid") = std::nullopt,
+         python::arg("choice"),
```

Comment:
By the way, I'm thinking of renaming those things "variant" rather than "choice"; the latter really isn't an ideal term, but was something I used just because it's used in the .json files. "variant" perhaps isn't ideal either, as it could be confused with std::variant. But it makes sense, on the other hand: variant is a std::variant. So, a user know what s/he's dealing with. Or, maybe there's a completely different name we should consider using.

Reply:
I'm not a fan of choice either. We can leave it for now but we should change it before we generate a full standard compliant interface.